web/src/games/secretDracula/board.tsx

```diff
@@ -553,4 +556,13 @@ export class Board extends React.Component<IBoardProps> {
       return;
     };
   }
+
+  _getGameOver() {
+    // Online game
+    if (this.props.ctx.gameover.win == true) {
```

Comment:
By looking at this implementation, I suspected this code wouldn't work, and it seems like thats the case:
![image](https://user-images.githubusercontent.com/1034631/99215199-17e6c500-2787-11eb-8bfc-74cec74503a3.png)
I will fix it for you though, should be a simple fix.